stable/appmesh-prometheus/templates/config.yaml

```diff
@@ -116,7 +100,16 @@ data:
         target_label: __metrics_path__
         replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
 
-    # scrape config for pods
+      # Exclude high cardinality metrics
+      metric_relabel_configs:
+      - source_labels: [__name__]
+        regex: (container|machine)_(cpu|memory|network|fs)_(.+)
+        action: keep
+      - source_labels: [__name__]
+        regex: container_memory_failures_total
```

Comment:
Why is this any different than other container metrics?

Reply:
Dropping that metric helps reduce the disk space usage dramatically. It a highly cardinality metric that we will not need in  any App Mesh dashboards.